iron-list.html

```diff
@@ -209,6 +216,34 @@
       },
 
       /**
+       * When true, `iron-list` will render the items in a fixed-width grid.
+       * If true the `itemWidth` property will also need to be set
+       */
+      grid: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true
+      },
+
+      /**
+       * When `grid` is true this will define the width of each item in the
+       * grid. This plus the value of `itemOffset` will determine how many
+       * items are rendered per row
+       */
+      itemWidth: {

```

Comment:
I was just implementing an optional `itemWidth` thing and totally had a change of heart.

I think that CSS is totally the right way to go for the positioning and the optional `itemOffset` also seems like a lot of overhead to introduce instead of just doing the nested div approach.

I am going to strip out the extra props and will update the PR.